app/controllers/api/v1/users_controller.rb

```diff
@@ -3,11 +3,7 @@ class Api::V1::UsersController < Api::BaseController
   before_action :check_ownership, only: %i[show update info]
```

Comment:
since we are removing `info` lets remove it from this list

Reply:
@nronas Done